build/release-tars/BUILD.bazel

```diff
@@ -168,6 +168,23 @@ pkg_tar(
     }),
 ) for arch in archs] for os, archs in CLIENT_PLATFORMS.items()]
 
+# This tarball contains the actual kubectl-cert-manager.tar.gz
+# This is meant to transport that file inside the build system
+# later to be extended with more requirements to use Krew.
+[[pkg_tar(
+    name = "krew-kubectl-cert-manager-%s-%s" % (os, arch),
```

Comment:
As noted above, I think this should be `cert-manager-kubectl-cert_manager-%s-%s` (regardless how odd that looks 😅)